content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -631,15 +622,15 @@ _secrets_ folder, your _identity.key_, your _secret.key_, and your
 _secret.key.not-so-secret_ file.
 
 [literal]
-+- identity.key   (RSA key pair that identifies an instance)
  +- secret.key     (used for various secure Jenkins operations)
  +- secret.key.not-so-secret  (used for validating _$JENKINS_HOME_ creation date)
  +- userContent    (files served in https://server/userContent/)
- +- secrets        (directory for the secret+key decryption)
+ +- secrets        (root directory for the secret+key for credential decryption)
```

Comment:
The secrets directory is used for more than credential decryption.  Many other secrets are decrypted using information stored in the secrets directory.
```suggestion
 +- secrets        (directory for the secret+key decryption)
```